client/src/actions/index.js

```diff
@@ -52,3 +52,14 @@ export const addFriend = data => async dispatch => {
     dispatch(addFriendFailure(err));
   }
 };
+
+export const updateFriend = (id, data) => async dispatch => {
```

Comment:
I really like the async await approach to the actions. Thanks for sharing that and the other libraries you use with us at the stand up. You're doing a great job Ashwin. Keep it up!